docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to Splunk Connect for Syslog
+
+Splunk Connect for Syslog is an open source packaged solution for 
+getting data into Splunk using syslog-ng (OSE) and the Splunk 
```

Comment:
Spell out “Open Source Edition” for first mention along with “OSE” abbreviation, then use abbreviation after that

Reply:
done